### PR TITLE
Add @Totktonada as codeowner for config

### PR DESCRIPTION
**Pull request by Totktonada**
I want to receive review request notifications about the changes and let people know who is responsible to accept them.

### CODE REVIEW

**Review by igormunkin — APPROVED** [review]
I personally have no objections (and even glad we're moving to splitting the sources by "maintainers"). However, it looks better to introduce `@config` group to be notified for the changes, and hence manage the working group there. Anyway, I leave the final decision on everything written above to @kyukhin.

**Review by kyukhin — APPROVED** [review]

**Comment by Totktonada** [comment]
@igormunkin @tarantool/config would mean all the developers that're involved into config's development. So, it should be @tarantool/config-maintainers with just me in the group :)

I don't mind, but I see no much difference from assigning me directly. I'll do, if Kirill decides that it is a more proper way.

**Review by ylobankov — APPROVED** [review]

**Pull request merged**
